cmake/DownloadDoubleConversion.cmake

```diff
@@ -0,0 +1,31 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+## Download Double Conversion library. 
+## SOURCE_DIR is typically the cmake source directory
+function(download_doubleconversion SOURCE_DIR BUILD_DIR)
+
+  ExternalProject_Add(
+    doubleconversion-proj
+    PREFIX "${BUILD_DIR}/dependencies"
+    GIT_REPOSITORY "https://github.com/google/double-conversion.git"
+    GIT_TAG "master"
```

Comment:
pin a version? "master" can break things?

Reply:
Their tags are over a year old and there are various bug fixes along the way. Since the docs for GIT_TAG say you can use a tag, branch, or commit I can use the latest commit with a comment that we're pinning against a current version of master. Could still be buggy -- I realize but we could re-evaluate at a later time.